src/components/SpecialResolution/CreateSpecialResolution.vue

```diff
@@ -341,8 +361,11 @@ export default class CreateSpecialResolution extends Mixins(DateMixin) {
     return this.yyyyMmDdToDate(this.getCurrentDate)
   }
 
-  /** The maximum date that can be entered (today). */
-  private get signatureDateMax (): Date {
```

Comment:
Getters don't need to be private since they could be called externally without issue.